src/components/NetworkDebuggerModal/index.js

```diff
@@ -69,7 +71,7 @@ const styles = StyleSheet.create({
   },
   content: {
     fontSize: 9,
-    fontFamily: 'Gilroy-Bold',
+    // fontFamily: 'Gilroy-Bold',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The commented out line `// fontFamily: 'Gilroy-Bold',` should be removed if it's not needed, to keep the code clean and maintainable.
```diff
- // fontFamily: 'Gilroy-Bold',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->